src/main/java/com/minecolonies/coremod/research/GlobalResearch.java

```diff
@@ -706,6 +706,10 @@ private ItemStack parseIconItemStacks(final String icon)
             try
             {
                 count = Integer.parseInt(iconParts[2]);
+                if(count > 127)
+                {
+                    count = 127;
+                }
```

Comment:
Shouldn't we limit that here rather to Stacksize? or maybe even maxStackSize?

Reply:
If we're being strict on that, I can limit it that heavily.  But these are very explicitly icons, and it seems like it's reasonable to have things like a pick 5 icon.  Only reason I didn't just pass the ItemIcon directly was to avoid potential sidedness issues.  

Or I can even just get rid of the whole behavior, since we're not using DRAW_ICONS at all right now.